Library/Homebrew/cmd/update.sh

```diff
@@ -215,7 +215,14 @@ merge_or_rebase() {
 
   trap reset_on_interrupt SIGINT
 
-  REMOTE_REF="origin/$UPSTREAM_BRANCH"
+  if [[ "$DIR" = "$HOMEBREW_REPOSITORY" && -z "$HOMEBREW_NO_UPDATE_CLEANUP" ]]
+  then
+    UPSTREAM_TAG="$(git tag --list --sort=-version:refname | head -n1)"
```

Comment:
Un a regular install tags aren't fetched, so `UPSTREAM_TAG` would be empty here.